xml/update-alternatives.xml

```diff
@@ -84,7 +83,7 @@
      <term>Link group</term>
      <listitem>
       <para>
-       A set of related symlinks that can be to be updated as a group.
+       A set of related symbolic links that can be updated as a group.
```

Comment:
I have a problem with this change. Although it's technically correct, the manpage talks about symlinks, not symbolic links. Additionally, the term is definied in the terminology list. I would suggest to phrase it like this:

```
A set of related symlinks (symbolic links) that ...
```